packages/manager/modules/pci/src/projects/project/kubernetes/add/translations/Messages_fr_FR.json

```diff
@@ -4,6 +4,8 @@
   "kubernetes_add_error": "Une erreur est survenue lors de la création de votre cluster: {{ message }}",
   "kubernetes_add_loading": "La création du cluster est en cours",
 
+  "kubernetes_add_region_inprogress": "La region {{ name }} est en cours d'ajout ...",
+  "kubernetes_add_region_failed": "La region {{ name }} n'as pas pu être ajoutée: {{ message }}",
```

Comment:
```suggestion
  "kubernetes_add_region_failed": "La region {{ name }} n'a pas pu être ajouté: {{ message }}",
```

Reply:
It was a tmp message.
We applied another message.